server/amp/helpers/infinite-scroll.js

```diff
@@ -41,34 +50,42 @@ class InfiniteScrollAmp {
     return `${hostWithProtocol}/${s3Key}?format=webp&w=250`;
   }
 
+  async getInfiniteScrollList(storyId) {
+    const sourceType = this.infiniteScroll && this.infiniteScroll.source;
```

Comment:
If user passes `infiniteScroll` obj without `source` this will store an unexpected value. Hence use lodash get to get the source.
Avoid using `&&` in such cases